components/brave_wallet/browser/eth_json_rpc_controller.cc

```diff
@@ -306,6 +307,8 @@ void EthJsonRpcController::OnGetTransactionReceipt(
 // static
 void EthJsonRpcController::RegisterProfilePrefs(PrefRegistrySimple* registry) {
   registry->RegisterListPref(kBraveWalletCustomNetworks);
+  registry->RegisterStringPref(kBraveWalletCurrentChainId,
```

Comment:
we also need a static function to clear the prefs when `KeyringController::Reset()`

Reply:
done